hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractS3ATestBase.java

```diff
@@ -60,6 +60,9 @@ public void setup() throws Exception {
     // filesystems which add default configuration resources to do it before
     // our tests start adding/removing options. See HADOOP-16626.
     FileSystem.getLocal(new Configuration());
+    // instantiate an S3A FS here here to force deprecated key load through the
+    // static initializers. See: HADOOP-17385
+    new S3AFileSystem();
```

Comment:
I think it would be cleaner if we could run the static initializers instead of creating an object without storing the reference. I know that there's a comment there stating that this is required because of a jira, but I don't see the point why would we create an object just to trigger side effects.

Reply:
I was about to say I don't know a way to trigger instantiation of static class initializers except by a new() call, but actually we can invoke any static method. How about I add a new static `initializeClass()` call and load it.

FWIW, what I'd really like is to be able to disable forced reload of configs, but I can't see a way to do that which wouldn't run the risk of damage